test/test_cubed_spheres.jl

```diff
@@ -20,6 +22,116 @@ using Oceananigans.Models.HydrostaticFreeSurfaceModels: VerticalVorticityField
 @inline extrinsic_vector_y_component(i, j, k, grid, uₑ, vₑ) =
     @inbounds intrinsic_vector(i, j, k, grid, uₑ, vₑ)[2]
 
+function kinetic_energy(u, v)
+    ke = Field(0.5 * (u * u + v * v))
+    return compute!(ke)
+end
+
+function test_vector_rotation(grid)
+    u = XFaceField(grid)
+    v = YFaceField(grid)
+    
+    # Purely zonal flow.
```

Comment:
not for all grids right?

Reply:
right, it's in the extrinsic frame. I have changed the comment